plugin/src/main/java/org/apache/jackrabbit/filevault/maven/packaging/VaultMojo.java

```diff
@@ -111,13 +111,31 @@
     /**
      * The directory containing the content to be packaged up into the content
      * package.
+     *
+     * This property is deprecated; use jcrRootSourceDirectory instead.
      */
-    @Parameter(
-            defaultValue = "${project.build.outputDirectory}",
-            required = true)
+    @Deprecated
+    @Parameter
     private File builtContentDirectory;
 
     /**
+     * The directory that contains the jcr_root of the content. Can be specified as comma separated list of paths.
```

Comment:
To me it is not clear from the comment how multiple entries are evaluated? All are merged in the final content-package or only the first existing directory is considered. It would be good to clarify the javadoc here.